public/js/index.js

```diff
@@ -825,6 +825,8 @@ function initWikiForm() {
                 "link", "image", "table", "horizontal-rule", "|",
                 "clean-block", "preview", "fullscreen"]
         })
+        var inputfield = simplemde.codemirror.getInputField();
+        $(inputfield).addClass("js-quick-submit");
```

Comment:
I think these two lines can be simply changed to:
```
$(simplemde.codemirror.getInputField()).addClass("js-quick-submit");
```

Reply:
Good point. I have pushed an update to merge the two lines.